backend/Repositories/SightingRepo.cs

```diff
@@ -18,6 +18,14 @@ public SightingRepo(WhaleSpottingDbContext context)
         {
             _context = context;
         }
+    
+        public IEnumerable<Sighting> GetApprovedSightings()
+        {
+            // return _context.Sightings
+            //     .Include(s => s.ConfirmationStatus);
+            
+            return _context.Sightings.Include(s => s.ConfirmationStatus).Where(s.ConfirmationStatus == "Approved");
```

Comment:
Format to different lines, and also `ConfirmationStatus` is now an enum
```suggestion
            return _context
                .Sightings
                .Where(s => s.ConfirmationStatus == ConfirmationStatus.Approved);
```